lib/agent0/agent0/interactive_fuzz/fuzz_present_value.py

```diff
@@ -118,14 +121,37 @@ def fuzz_present_value(
             case HyperdriveActionType.CLOSE_SHORT:
                 maturity_time, open_trade = next(iter(agent.wallet.shorts.items()))
                 trade_event = agent.close_short(maturity_time=maturity_time, bonds=open_trade.balance)
+            case _:
+                raise ValueError(f"Invalid {trade_type=}")
```

Comment:
I believe these trades are not getting checked in invariances, this for loop is simply opening trades. A quick scan shows there are indeed checks against these trades

Reply:
yeah I just caught that too. one sec